### ISSUE TITLE
Xcode 13 warnings modules/ecdh + modules/recovery file not found

### ISSUE THREAD

**Issue by simonmcl** [Boilertalk/secp256k1.swift#13]
Trying to update my Swift package to use Xcode 13. Getting several warnings about missing files. Can someone add these?

![Screenshot 2021-10-07 at 11 33 13](https://user-images.githubusercontent.com/8058386/136368441-a8d13225-c104-4fd3-af0e-78b872088a0d.png)

**Comment by koraykoska** [comment]
Is this still an issue?

**Comment by simonmcl** [comment]
@koraykoska just updated to `0.1.6` unfortunately no its not resolved. I'm not very familiar with the underlying library or how its stitched together. Looking at the package.swift it contains multiple references to paths that don't exist in the C codes submodule:

under `exclude`
```
"secp256k1/src/modules/ecdh/tests_impl.h",
"secp256k1/src/modules/recovery/tests_impl.h",
```
and, under `sources`
```
"secp256k1/modules/ecdh",
"secp256k1/modules/recovery"
```

There is no `modules` folder. Possibly `modules` was removed and the code moved elsewhere. The previous PR was possibly just re-adding the code but not dealing with the broken paths.

Can you double check i'm not missing something?

**Comment by koraykoska** [comment]
@simonmcl Fixed in version 0.1.7. Please upgrade. Web3.swift will get an update either today or tomorrow.

Thanks for highlighting this.
